step_definitions/test_steps.js

```diff
@@ -17,7 +17,7 @@ Given("user visits url {string}", async function (url) {
 // =========================================== Steps ==================================================================
 
 Then('click {string} link', async (text) => {
-	await clickElementWithSelector(scope.page, 'a', text)
```

Comment:
@sercanvirlan why are we removing "a" tag. The helper method does match the elements by selector and the text value (innerHTML). Since this test step only tests "links" it needs to keep the <a> tag selector here.

Reply:
If anchor elems contain another node other than TextNode for example span this is not working. I may create anoyher step definition rather than changing this btw.